stories/TransLine.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import { Story, Meta } from '@storybook/react'
+import { TransLine as TransLineComponent, TransLineProps } from '../components/TransLine'
+
+export default {
+  title: 'Component/Atoms/Trans Line',
+  component: TransLineComponent,
+} as Meta
+
+const Template: Story<TransLineProps> = (args) => <TransLineComponent {...args} />
+
+export const TransLine = Template.bind({})
+TransLine.args = {
+  i18nString: 'work-in-progress.message',
+  links: ['https://navapbc.com', 'https://edd.ca.gov'],
```

Comment:
I'm a little confused by what's passed in vs what I see here!

 - input: i18nstring with one link substition, two links
 - output: text string with one line, the _latter_ one

is that expected behavior to get the second link/last link? why are we passing in two?

Reply:
ahah, its taking the link from the index referenced in the key (`1`), so that makes sense there. Is the reason we're passing in two links just to get the positional syntax correct for the i18n string? I think only passing in one item and correcting the index might be cleaner

Reply:
> ahah, its taking the link from the index referenced in the key (`1`), so that makes sense there. Is the reason we're passing in two links just to get the positional syntax correct for the i18n string? 

Yes, that's correct. 

> I think only passing in one item and correcting the index might be cleaner

I agree, but I didn't want to introduce a translation string just for the sake of a story, which is why I re-used `work-in-progress.message`. I'm not sure what is the best path forward here. Thoughts?

Reply:
How about an explainer comment in the code, and maybe name the first URL something obviously placeholder-y, like `example.com/spacer`?